data/js/duckduckgo.js

```diff
@@ -1,4 +1,16 @@
-self.port.emit('loaded');
+// grab the ATB param from the html element's class
+function getATBParam() {
```

Comment:
@bsstoner I am not sure if this is the best way of doing this in vanilla JavaScript but it is certainly the cleanest one I found.